src/main/java/github/scarsz/discordsrv/DiscordSRV.java

```diff
@@ -721,19 +726,42 @@ private List<InetAddress> lookupPublic(String host) throws UnknownHostException
         dispatcher.setMaxRequestsPerHost(20); // most requests are to discord.com
         ConnectionPool connectionPool = new ConnectionPool(5, 10, TimeUnit.SECONDS);
 
-        OkHttpClient httpClient = new OkHttpClient.Builder()
+        String proxyHost = config.getString("ProxyHost").trim();
+        String proxyPort = config.getString("ProxyPort").trim();
+        final String authUser = config.getString("ProxyUser").trim();
+        final String authPassword = config.getString("ProxyPassword").trim();
```

Comment:
```suggestion
        String proxyHost = config.getString("ProxyHost").trim();
        String proxyPort = config.getString("ProxyPort").trim();
        String authUser = config.getString("ProxyUser").trim();
        String authPassword = config.getString("ProxyPassword").trim();
```
you don't need `final`s here